### PR TITLE
Switch from balenaCI to flowzone

### PR DESCRIPTION
**Pull request by Page-**
Change-type: patch

### Hey there, appreciated contributor!

Have you taken a look at the [contribution guidelines](../CONTRIBUTING.md) for this project? To enable our CI to work correctly and version the releases of the project, you need to ensure your commit messages contain the required information [detailed here](../CONTRIBUTING.md). Thank you!

### CODE REVIEW

**Comment by Page-** [comment]
@balena-ci I self-certify!

**Comment by klutchell** [comment]
@Page- my draft PR with the buildx bake changes will skip docker tests if a docker-compose test file does not exist. That should unblock this repo that only supports armv7.

File: .github/workflows/flowzone.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+      - "main"
+      - "master"
+
+jobs:
+  flowzone:
+    name: Flowzone
+    uses: product-os/flowzone/.github/workflows/flowzone.yml@master
+    secrets: inherit
+    with:
+      balena_slugs: balenalabs/balena-sense
```
Comment (klutchell) [inline .github/workflows/flowzone.yml:16]: Looks like this fleet does not exist, or the token we are using doesn't have access?

Comment (Page-) [reply .github/workflows/flowzone.yml:16]: Maybe it doesn't but it's what balenaCI would have pushed to for the `type: balena` repo

**Comment by klutchell** [comment]
I didn't know balenaCI could do balena push?

**Comment by maggie0002** [comment]
Closing as already implemented: https://github.com/balena-labs-projects/balena-sense/blob/master/.github/workflows/flowzone.yml

**Pull request closed without merging**